scripts/paper/README.md

```diff
@@ -0,0 +1,14 @@
+# Creating symlinks to files on network drive
+Run the following python script to create symlinks to CZI files on the network drive:
```

Comment:
There should be some note that this is for folks internal at allen

Reply:
Good point. Note added.